.github/workflows/main.yml

```
@@ -38,7 +38,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.8", "3.12"]
+        python-version: ["3.8", "3.12","3.13.0-rc.2"]
```

Thanks! `3.13-dev` will get the latest prerelease without needing to update here:

```suggestion
        python-version:
          - "3.8"
          - "3.12"
          - "3.13-dev"
```